rpcs3/Emu/Cell/PPUModule.h

```diff
@@ -273,6 +276,9 @@ class ppu_module_manager final
 	static const ppu_static_module sys_libc;
 	static const ppu_static_module sys_lv2dbg;
 	static const ppu_static_module static_hle;
+
+private:
+	inline static std::unordered_map<std::string, ppu_static_module*> staticModuleMap;
```

Comment:
we prefer snake_case over CamelCase especially in places where the rest of the code is already snake_case.